orionsdk/swisclient.py

```diff
@@ -39,6 +39,9 @@ def read(self, uri):
     def update(self, uri, **properties):
         self._req("POST", uri, properties)
 
+    def bulkupdate(self, uri, body):
```

Comment:
I think this method signature can be improved. Instead of having an unused `uri` parameter and requiring the caller to know the correct format for the `body` parameter, how about this?

```python
def bulkupdate(self, uris, **properties):
```

Then the bulkupdate method could assemble the correct body structure given the list of uris and properties dict.

Reply:
That looks good to me!

Reply:
So was I adjusting my PR or were you going to make the change?